openhcl/virt_mshv_vtl/src/processor/snp/mod.rs

```diff
@@ -1422,7 +1422,7 @@ impl UhProcessor<'_, SnpBacked> {
                 let interruption_pending = vmsa.event_inject().valid()
                     || SevEventInjectInfo::from(vmsa.exit_int_info()).valid();
                 let exit_message = self.runner.exit_message();
-                let emulate = match exit_message.header.typ {
+                let real = match exit_message.header.typ {
```

Comment:
don't we need to fix this in the future so we don't look at the hypervisor message since we don't trust the host? is there something tracking that?